File: ABAModelDiversity/API/Tables/CompanyProfiles.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Tables
{
  [Table("AD_CompanyProfiles")]

  public class CompanyProfiles
  {
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid CompanyProfileID { get; set; }
    public string Firmname { get; set; }
    public string Firmhead { get; set; }
    public DateTime Datecomp { get; set; }
    public string Srcname { get; set; }
    public string Srctitle { get; set; }
    public string Srcemail { get; set; }
    public int Totalfw { get; set; }
    public int Totalusfw { get; set; }
    public int Sizecat { get; set; }
    public string Firmown { get; set; }
    public string Catown { get; set; }
    public string Firmcert { get; set; }
  }

}

File: ABADiversity/src/app/entities/companyprofiles.ts
export interface CompanyProfiles {
    companyProfileID?:string;
    firmID?:string;
    firmname?:string;
    firmhead?:string;
    datecomp?:Date;
    srcname?:string;
    srctitle?:string;
    srcemail?:string;
    totalfw?:number;
    totalusfw?:number;
    sizecat?:number;
    firmown?:string;
    catown?:string;
    firmcert?:string;
}

export interface Certificates{
    certificateID ?:string,
    companyProfileID?:string;
    name?:string,
    dateCert?:Date
}

export interface Years{
  companyProfileID?:string;
  year:number;   
}